src/Components/ErrorPopUp/index.js

```diff
@@ -0,0 +1,38 @@
+import React, { useState } from 'react';
+import styles from './style.module.css';
+
+function ErrorPopUp(props) {
+  const [show, setShow] = useState(true);
+  const [errorMessage, setErrorMessage] = useState('Something went wrong while processing the application status update. Please try again.');
+  const disapear = () => setShow(false);
+  const { apiErrorMessage } = props;
+
+  if (apiErrorMessage) setErrorMessage(apiErrorMessage);
```

Comment:
even though it does look a bit cleaner writing your conditionals like this and its not a huge deal either way, but I think its best practice to use curly braces. 
i.e. `if (apiErrorMessage) { setErrorMessage(apiErrorMessage); }`